packages/venia-ui/lib/components/Header/accountTrigger.css

```diff
@@ -0,0 +1,26 @@
+.root {
+    align-items: center;
+    display: grid;
+    /* The full height of the site header minus (box shadow height * 2). */
+    height: calc(5rem - 8px);
```

Comment:
I'm surprised `height` isn't a token.

Reply:
I created a separate ticket for this one 👍